.semaphore/semaphore.yml

```diff
@@ -101,7 +101,7 @@ blocks:
           commands:
             - cd users
             - cache restore linter
-            - './bin/golangci-lint run'
+            - './bin/golangci-lint run --no-config --disable-all --enable goimports,gofumpt,gci,gosimple,errcheck,govet,unused,goconst,godot,gofumpt --timeout 3m'
```

Comment:
We have added the CI linter config to the root, so no need for these changes.

Reply:
I don't understand, when we add the linter to root we did not enable the different options

Reply:
I only added gofumpt to check for fomatting.

Reply:
Understood